libraries/botbuilder-dialogs-adaptive/package.json

```diff
@@ -28,7 +28,7 @@
     }
   },
   "dependencies": {
-    "@microsoft/recognizers-text-suite": "1.1.4",
+    "@microsoft/recognizers-text-suite": "^1.3.0",
```

Comment:
I recall this library is used in more than one package - is it important that we use the same version across all our packages?

Reply:
There might be some bugs in the old version of this package, so I have to update the package or the test cases I ported from C# will fail.

Reply:
After searching across the repo, I found this package was used by the following packages:
* adaptive-expressions
* botbuilder-ai
* botbuilder-dialogs
* botbuilder-dialogs-adaptive
* testbot

Reply:
Do we need it to be consistent across these packages?

Reply:
I think we'd better change the version to `^1.3.0` over all packages.

Reply:
We’ve had past difficulties with this collection of dependencies in JS and .NET. They tend to not follow semver... we should consult with @EricDahlvang and do a hand-off of this dependency’s validation process for uptake into the JS SDK.

Reply:
@stevengum  https://github.com/microsoft/botbuilder-js/pull/1934

Reply:
@joshgummersall @stevengum @EricDahlvang I would change the version back to `1.1.4` to align with other packages, and I have to made some changes on the test cases to let the tests pass.